includes/functions.inc.php

```diff
@@ -328,7 +328,7 @@ function arrayClean(&$array) {
 function stripAssocArray($values) {
 	foreach ($values as $key => $value) {
 		if (is_scalar($value)) {
-			$values[$key] = strip_tags($values[$key]);
+			$values[$key] = html_entity_decode(strip_tags($values[$key]), ENT_QUOTES, Config::getVar('i18n', 'client_charset'));
```

Comment:
The OAI interface is fixed to use UTF8, regardless of what the client charset is set to, per the OAI-PMH spec.